extra/aur/sunpaper.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
@hexive  we probably have to extract those two files in separate PR, what do you think? I'm having issues getting PKGBUILD to pick up the changes